### DISCUSSION TITLE
Support of Budgie on FreeBSD

### DISCUSSION THREAD

**Discussion by OlivierDuchateau** [BuddiesOfBudgie/budgie-desktop#326] [General]
The Budgie desktop is now [available](https://cgit.freebsd.org/ports/commit/?id=1ccad74cc8c19d8a0fa0ab573604db5c3484d90d) on FreeBSD. I wonder, if you think is useful to add instructions how to install on this system.

I maintain this documentation on my [own repository](https://codeberg.org/olivierd/freebsd-ports-budgie/wiki), but I would like something more official.

**Comment by JoshStrobl** [comment]
Hey @OlivierDuchateau, a PR would be welcome to https://github.com/BuddiesOfBudgie/docs

We have a Getting Budgie page on our new [documentation center](https://docs.buddiesofbudgie.org) and I think FreeBSD instructions would be great there! Would also need to change "are available across a wide variety of Linux-based operating systems." at the top of it to something like "are available across a wide variety of operating systems." so it's not "Linux-based" anymore :)

The GitHub wiki will be nuked once everything has been ported over.
